index.js

```diff
@@ -12,15 +12,20 @@ module.exports = {
     this._super.included.apply(this, arguments);
 
     var hostApp = this._findApp(app);
-    this._regeneratorAlreadyIncluded =
+
+    var babelOptions = (hostApp.options && hostApp.options.babel) || {};
+    var emberCLIBabelOptions = (hostApp.options && hostApp.options['ember-cli-babel']) || {};
+
+    var babelInstance = this.addons.filter(function(addon) { return addon.name === 'ember-cli-babel'; })[0];
```

Comment:
This is node 4+. You don't need to do `filter + [0]` anymore, you can use `find`.